gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/factories/KubernetesObjectFactory.java

```diff
@@ -102,6 +115,20 @@ private V1Secret createSecretFromValues(final Object helmValues, final Gaffer ga
                 .build();
     }
 
+    private GafferSpec addOpenShiftHelmValues(final GafferSpec helmValues) {
+        helmValues.putNestedObject("edge", "ingress", "annotations", "route.openshift.io/termination");
```

Comment:
This is the config for openshift instead of putting it into gaffer docker? 👍

Reply:
Correct